src/MongoDB/Query.c

```diff
@@ -268,33 +268,33 @@ static bool php_phongo_query_init(php_phongo_query_t *intern, zval *filter, zval
 	}
 
 	PHONGO_QUERY_OPT_BOOL("allowPartialResults", options, "allowPartialResults")
-		else PHONGO_QUERY_OPT_BOOL("allowPartialResults", options, "partial");
+	else PHONGO_QUERY_OPT_BOOL("allowPartialResults", options, "partial");
```

Comment:
This is a silly one. I'm putting the ``else...`` on the same line to avoid this nonsense.

Reply:
Except that that doesn't work...

Reply:
What happens? It injects a line break anyway? I don't really mind, as `else` still stands out from `PHONGO` and shouldn't hurt readability too much.

Is the trailing semicolon necessary? All of these macros are just `if` blocks, so I assume not. Perhaps we can delete it so that the lines starting with `PHONGO` and `else` both end consistently.

Reply:
It indeed injects a line break anyway. clang-format really likes things that look like statements end with a statement separator (``;``), so IMO, we should have one there.